include/nil/crypto3/math/domains/detail/basic_radix2_domain_aux.hpp

```diff
@@ -108,8 +140,13 @@ namespace nil {
                     if (omega_cache == nullptr) {
                         std::vector<typename FieldType::value_type> omega_powers;
                         create_fft_cache<FieldType>(a.size(), omega, omega_powers);
+if (omega_powers.size() == 0)
+    throw "We have a problem here.";
```

Comment:
If we use throw - when we need to implement catch somewhere... or just use assert.

Reply:
Thanks, this was a leftover from my debugging section, it does not belong here. Removed.